backend.native/tests/interop/objc/smoke.h

```diff
@@ -248,4 +248,28 @@ extern BOOL customStringDeallocated;
 @property int value;
 - (int)instanceMethod;
 + (int)classMethod:(int)first :(int)second;
-@end;
\ No newline at end of file
+@end;
+
+// [KT-36067] cinterop tool fails when there is a structure member named Companion
+
+struct EnumFieldMangleStruct {
+	enum {Companion, Any} smth;
+};
+struct EnumFieldMangleStruct enumMangledStruct = { Any };
+
+struct MyStruct {
+	int Companion;      // simple clash
+	int _Companion;
+	int $_Companion;
```

Comment:
Since mangling scheme changed, these cases should be updated as well.

Reply:
Last three members are examples of name which are _not affected_ by mangling scheme. The case which demostrated "secondary clash" avoidance is removed now, as it is not supported in the simple scheme.